docs/content/gds/admin/configuration/trtl.en.md

```diff
@@ -1,7 +1,98 @@
+
+### Replica Config
+
+TrtlDB is a globally replicated database and the replica config provides metadata about the TrtlDB instance for replication purposes while also setting parameters for replication. Replication is configured as follows:
+
+| EnvVar                       | Type     | Default | Description                                                                                                           |
+|------------------------------|----------|---------|-----------------------------------------------------------------------------------------------------------------------|
+| TRTL_REPLICA_ENABLED         | bool     | true    | If false, disables replication so that the trtldb acts as a single node.                                              |
+| TRTL_REPLICA_PID             | uint64   |         | The precedence ID of the node (must be unique in the system) - lower IDs have precedence in consistency tie breakers. |
+| TRTL_REPLICA_REGION          | string   |         | The region that the replica is assigned to for provenance and geographic compliance.                                  |
+| TRTL_REPLICA_NAME            | string   |         | A unique name that identifies the replica name across peers.                                                          |
+| TRTL_REPLICA_GOSSIP_INTERVAL | duration | 1m      | The mean interval between gossip (synchronization) sessions between trtl peers.                                       |
+| TRTL_REPLICA_GOSSIP_SIGMA    | duration | 5s      | The standard deviation of the jittered interval between gossip sessions.                                              |
+
+If `TRTL_REPLICA_ENABLED` is `true` then `TRTL_REPLICA_PID`, `TRTL_REPLICA_REGION`, and `TRTL_REPLICA_NAME` are required to identify the unique replica in the system.
+
+Replication occurs with bilateral anti-entropy, meaning that after a jittered interval, each replica randomly selects a peer to synchronize with. This is also referred to as a gossip protocol. The `TRTL_REPLICA_GOSSIP_INTERVAL` and `TRTL_REPLICA_GOSSIP_SIGMA` describe a normal distribution of randomly selected synchronization intervals, providing jitter so that the network is not bursty.
+
+### Replica Identification Strategy
+
+When deployed in on a kubernetes cluster, a trtl process running in a container in a pod must identify itself to determine what replica it is so that it can configure itself correctly (particularly if the replica is part of a stateful set). The replica strategy configuration defines how a trtl process bootstraps itself as follows:
```

Comment:
```suggestion
When deployed in a kubernetes cluster, a trtl process running in a container in a pod must identify itself to determine what replica it is so that it can configure itself correctly (particularly if the replica is part of a stateful set). The replica strategy configuration defines how a trtl process bootstraps itself as follows:
```